prac_05/wimbledon.py

```diff
@@ -0,0 +1,46 @@
+"""
+Wimbledon
+Estimate: 40 minutes
+Actual:   40 minutes
+"""
+
```

Comment:
It is better when you create a global constant (INDEX-COUNTRY=1 ) 
(INDEX-CHAMPION=2)

Reply:
Thank you. I'll fix it.